packages/server/graphql/types/NewMeetingPhaseTypeEnum.ts

```diff
@@ -32,7 +33,9 @@ const NewMeetingPhaseTypeEnum = new GraphQLEnumType({
     [DISCUSS]: {},
     SUMMARY: {},
     SCOPE: {},
-    ESTIMATE: {}
+    ESTIMATE: {},
+    // team prompt
+    [RESPONSES]: {}
```

Comment:
+1 i need to do a better job about code comments on deprecated patterns!

In the future, where everything is better, this GraphQLEnumType will be populated by the Postgres enum type.
The relay-compiler will generate client-side enums from that (in the __generated__ folder) and our front-end components can use the types from there.

That is to say, could you just call this `RESPONSES: {}` and remove that value from the client/utils/constants?

Reply:
Sure thing, done! And it makes sense, thank you for pointing that out 👌